spec/spec_helper.rb

```diff
@@ -63,4 +62,15 @@
   config.filter_run_excluding :broken => true
 
   config.fail_fast = false
+
+  config.before(:all) {
+    if self.class.metadata[:vcr]
+      VCR.insert_cassette("#{self.class.metadata[:description]}/all")
+    end
+  }
+  config.after(:all) {
+    if self.class.metadata[:vcr]
```

Comment:
Can we make this behaviour default with an opt-out instead of opt-in? So we don't have to manually  add the :vcr meta flag to all the spec files...

Reply:
Manually marking tests as vcr capable is exactly what I wanted to achieve ;-) In my opinion we should add tests to vcr incrementally, properly checking for possible sensitive data. In addition it makes no sense to have it enabled for tests which has no cassete recorded, since our test runtime environment is wiped after each suite run, in such case it brings only overhead.